### PR TITLE
Move text_content and file_path inside ContentAnalysisRequest.

### PR DESCRIPTION
**Pull request by nancy-lan-xiao**
Move text_content and file_path inside ContentAnalysisRequest. 

Fix typo in the comment.

### CODE REVIEW

**Review by eric-waldman — DISMISSED** [review]
LGTM

File: proto/content_analysis/sdk/analysis.proto [hunk 1]
```diff
@@ -74,10 +67,19 @@ message ContentAnalysisRequest {
...
   repeated string tags = 11;
 
+  // The data to analyze in the request. These fields are mutually exclusive,
+  // only one will be set in any content analysis request.
+  message ContentData {
+    // The text content to analyze.
+    optional string text_content = 1;
+    // The full path to the file to analyze. The path is expressed in a platform dependent way. 
+    optional string file_path = 2;
+  }
```
Comment (rogerta) [inline proto/content_analysis/sdk/analysis.proto:77]: I was thinking we would remove the ContentData message and just put the two fields directly into the request.  Is there a reason why we'd want to keep the message?

Another option might be to make this change to connectors.proto in the chromium repo first, and bringing the changes here.  That way we can get a review by chromium owners first.

File: CMakeLists.txt [hunk 2]
```diff
@@ -115,4 +115,4 @@ target_link_libraries(agent PRIVATE cac_agent)
 # The demo client executable.
 add_executable(browser ./demo/client.cc)
 target_include_directories(browser PRIVATE ${BROWSER_INCLUDES})
-target_link_libraries(browser PRIVATE cac_browser)
+target_link_libraries(browser PRIVATE cac_browser)
```
Comment (rogerta) [inline CMakeLists.txt:118]: Leave the newline in?

**Review by rogerta — COMMENTED** [review]
Looking good Nancy.  What do you think about doing the review in chromium first?

**Comment by nancy-lan-xiao** [comment]
Code submitted via https://github.com/chromium/content_analysis_sdk/pull/32

**Pull request closed without merging**